Tangem/App/ViewModels/CardViewModel.swift

```diff
@@ -304,6 +303,30 @@ class CardViewModel: Identifiable, ObservableObject {
         )
     }
 
+    func validate() -> Bool {
+        if userWallet.hasBackupErrors == true {
```

Comment:
не знаю какой вариант лучше)
```suggestion
        if userWallet.hasBackupErrors ?? false {
```

Reply:
мне кажется тот что сейчас есть проще читается

Reply:
он единственное чем смущает - что странно булку сравнивать с `true` или `false`, но в целом так удобнее, если опционал)